optuna/samplers/_base.py

```diff
@@ -148,3 +148,12 @@ def reseed_rng(self) -> None:
         """
 
         pass
+
+    @staticmethod
+    def _raise_error_if_multi_objective(study: Study) -> None:
+
+        if len(study.directions) > 1:
+            raise RuntimeError(
```

Comment:
According to the [official document](https://docs.python.org/3/library/exceptions.html#RuntimeError), it seems that `RuntimeError` is an option chosen by process of elimination when any other error cannot be applied. I think `ValueError` is reasonable since the visualization functions have the same error, as you pointed. I will fix it to `ValueError`.